### ISSUE TITLE
Import Info Tooltip not Accessible

### ISSUE THREAD

**Issue by ElectronicBlueberry** [galaxyproject/galaxy#15383]
The import info addressed in #14857 is not accessible.
`aria-describedby` is being added on hover, but there is no way to access this information  without a mouse. I suggest adding a sr-only text

**To Reproduce**
1. Import a workflow
2. Try seeing which workflows are imported relying on text and keyboard only

**Expected behavior**
It is in some way communicated which workflows are imported, without needing to rely on mouse or icons.

**Additional context**
A solution might be to duplicate the label on the focusable workflow name
